ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -24,22 +26,22 @@
  */
 public class BlobPropertiesSerDe {
 
-  static final short Version1 = 1;
-  static final short Version2 = 2;
-  private static final short currentVersion = Version1;
-  private static final int Version_Field_Size_In_Bytes = 2;
-  private static final int TTL_Field_Size_In_Bytes = 8;
-  private static final int Private_Field_Size_In_Bytes = 1;
-  private static final int CreationTime_Field_Size_In_Bytes = 8;
-  private static final int Variable_Field_Size_In_Bytes = 4;
-  private static final int BlobSize_Field_Size_In_Bytes = 8;
+  static final short VERSION_1 = 1;
+  static final short VERSION_2 = 2;
+  private static final short CURRENT_VERSION = VERSION_2;
+  private static final int VERSION_FIELD_SIZE_IN_BYTES = Short.BYTES;
+  private static final int TTL_FIELD_SIZE_IN_BYTES = Long.BYTES;
+  private static final int PRIVATE_FIELD_SIZE_IN_BYTES = Byte.BYTES;
+  private static final int CREATION_TIME_FIELD_SIZE_IN_BYTES = Long.BYTES;
+  private static final int VARIABLE_FIELD_SIZE_IN_BYTES = Integer.BYTES;
+  private static final int BLOB_SIZE_FIELD_SIZE_IN_BYTES = Long.BYTES;
 
   public static int getBlobPropertiesSerDeSize(BlobProperties properties) {
```

Comment:
Not really related to the patch, but maybe rename this `getSerializedSize()` or `getSerializedBlobPropertiesSize()`